recipes/pyiron_database/meta.yaml

```
@@ -0,0 +1,51 @@
+{% set name = "pyiron_database" %}
+{% set version = "0.0.8" %}
+{% set python_min = "3.10" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 7577eada2ba9b8b7846f0336386060e1f8c5c99f6e632101608766ac8ea8b3ad
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python {{ python_min }}
+    - setuptools
+    - pip
+    - h5py
+    - versioneer
+    - toml
+    - pyiron_workflow
```

```suggestion
    - versioneer
    - toml
```
I think I need at least toml and versioneer to get the version correctly included. Actually, I saw on other packages in the pyiron org that the version is build in and versioneer is dropped from the run dependencies. I work to get it here in a similar fashion